### DISCUSSION TITLE
https://github.com/noah-nuebling/mac-mouse-fix/blob/version-3/LICENSE is not working.

### DISCUSSION THREAD

**Discussion by TeodoraMih** [noah-nuebling/mac-mouse-fix#1085] [Questions] [answered]
Hi,

We would need a link to EULA for the paid license of **Mac Mouse Fix v3.**
Looks like this link is no longer valid: [](https://github.com/noah-nuebling/mac-mouse-fix/blob/version-3/LICENSE)
Could you kindly guide me on where to find the licensing terms?

Kind regards,

**Comment by noah-nuebling** [comment, marked as answer]
Hi there! 

The licensing terms can be found here:

https://github.com/noah-nuebling/mac-mouse-fix/blob/master/License

Where did you find the dead link?

**Comment by TeodoraMih** [comment]
Hi Noah,

That broken link , it was mentioned in this discussion<https://github.com/KingIsulgard/iOS-Polynomial-Regression/issues/6> :
***@***.***

Anyway, thank you for clarification and for the correct link to EULA.

Best regards,

Teodora Mihalea
Specialist | SAM and Software Licensing, CT
***@***.***
Veeam Software<https://www.veeam.com/>
***@***.******@***.***>
